lib/linter-integration.js

```diff
@@ -31,13 +31,15 @@ class Linter {
     }
     this.linter.setMessages(messages.map(match => ({
       type: match.type || 'Error',
-      text: match.message || 'Error from build',
+      text: !match.message && !match.html_message ? 'Error from build' : match.message,
+      html: match.html_message,
```

Comment:
Sure, that sounds reasonable as long as it is documented. I'll make the necessary changes.